changes.py

```diff
@@ -310,32 +306,68 @@ def get_logs(src: t.List[str], finder: t.Callable, extractor: t.Callable) -> t.D
     return logs
 
 
-#
-def get_changelog_by_name(lb_type: str, lb_name: str, common_logs: t.Dict[str, common_log],
-                          spec_logs: t.Dict[str, spec_log]) -> t.List[str]:
-    changelog = common_logs[lb_type].changelog.copy()  # скопируем общую часть
-    if lb_name in spec_logs:  # и если есть частность
-        changelog += spec_logs[lb_name].changelog  # то и её добавим
+def get_full_log(lb_type: str, lb_name: str, common_logs: t.Dict[str, common_log],
+                 spec_logs: t.Dict[str, spec_log]) -> t.List[str]:
+
+    # скопируем общую часть
+    changelog = common_logs[lb_type].changelog.copy()
+
+    # и если есть частность
+    if lb_name in spec_logs:
+
+        # то и её добавим
+        changelog += spec_logs[lb_name].changelog
 
     return changelog
 
 
-# заполнялка одной записи результата
-def fill_row(row, lb_type, prev_v, new_v, common_logs, spec_logs):
-    res_row = {}
-    res_row['IE_XML_ID'] = row['IE_XML_ID'].replace(str(prev_v), str(new_v))
-    res_row['IE_NAME'] = IE_NAME_PATTERN.format(int_to_dotted_str(new_v))  # пример: Версия 9.8.2
+def replace_with_next_num(string: str, num: int) -> str:
+    """
+    Заменяет в исходной строке string все вхождения числа num на num + 1
+    """
+    return string.replace(str(num), str(num + 1))
+
+
+def formatted_dotted_next_num(num: int, pattern=IE_NAME_PATTERN) -> str:
```

Comment:
```suggestion
def format_with_dots(num: int, pattern=IE_NAME_PATTERN) -> str:
```

Передавать извне уже увеличенное значение (чтобы уменьшить зону ответственности функции)

Reply:
Исправлю.